README.md

```diff
@@ -6,6 +6,38 @@ Create a community of like minded authors to foster inspiration and innovation
 by leveraging the modern web.
 
 ---
+## Getting started 
+These instructions will get you a copy of the project up and running in your local machine for development and testing purposes.
+
+## Prerequisites
+- [Git](https://git-scm.com/download/)
+- [Python 3 and above](https://www.python.org/downloads/)
+- [PostgreSQL](https://www.postgresql.org/)
+
+## Installing
+- Create a working space in your local machine
+- Clone this [repository](https://github.com/andela/ah-django-unchained) `git clone https://github.com/andela/ah-django-unchained.git`
+- Navigate to the project directory
+- Create a virtual environment `python3 -m venv/path/to/new/virtual/environment`
+- Activate your virtual environment `source name_of_your_virtual_environment/bin/activate`
+- Install dependencies to your virtual environment `pip install -r requirements.txt`
```

Comment:
Please note requirements.txt uses django 1.10. Update the requirements.txt file or state in your readme we are using django 2.0

Reply:
Kwame I think the versions of dependencies are catered in the requirements.txt and the requirement.txt in this branch is not up to date but I think when the branch is merged it will be up to date because Maggy upgraded the Django version in the branch that she was working on.